com.unity.render-pipelines.core/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [7.5.3] - 2021-11-1
```

Comment:
iirc we had some validation tests issues in the past with dates with only one digit (like this one).
Please add a `0` to the date: `2021-11-01`

Reply:
Note: For all packages

Reply:
looking at the history, shouldn't the date/month be reversed as well? 
`2021-01-11`

Reply:
good point!